load/dependencies.go

```diff
@@ -9,22 +9,23 @@ import (
 
 type TxGenerator[T comparable] interface {
 	// GenerateTx returns a valid transaction.
-	GenerateTx(ctx context.Context) (T, error)
+	GenerateTx() (T, error)
```

Comment:
What's the motivation for removing the context here? Generating a transaction could likely depend on making some number of RPC requests, so it'd be good to have a context available to use in that case.

Reply:
This is changing to be a `GenerateAndIssueTx(context.Context)` anyway, so no worry on this, a context will be available.

Now the previous rationale was just that generating txs should be quick, and we don't need any IO (here or in geth-based-evms), so there was no need for a context. Something we could change further down the line though, of course.

Reply:
Yeah, for any interesting/complex transaction types in generic VMs, I think a context will be needed. 

For instance, when calling a smart contract, the only good way to estimate the gas needed is to use the `eth_estimateGas` RPC. I'd image in the future there could be similar cases in other VMs for things such as fetching UTXOs to spend, etc.

Reply:
I think we should probably leave the context in the interface for now to reduce the changes here, and potential changes needed down the line